app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -231,13 +231,27 @@ class ItemListFragment : Fragment(), ItemListView {
 
     override fun showToastNotification(@StringRes strId: Int) {
         assertOnUiThread()
+        val toast = setUpToast(strId = strId)
+        toast.show()
+    }
+
+    override fun showDeleteToastNotification(text: String) {
+        assertOnUiThread()
+        val toast = setUpToast(text = text)
+        toast.show()
+    }
+
+    private fun setUpToast(@StringRes strId: Int? = null, text: String? = null): Toast {
         val toast = Toast(activity)
+
         toast.duration = Toast.LENGTH_SHORT
         toast.view = layoutInflater.inflate(R.layout.toast_view, this.view as ViewGroup, false)
         toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, 0)
-        val v = toast.view.findViewById(R.id.message) as TextView
-        v.text = resources.getString(strId)
-        toast.show()
+
+        val view = toast.view.findViewById(R.id.message) as TextView
+        view.text = (text ?: resources.getString(strId!!)).plus(" deleted.")
```

Comment:
this ends up showing " deleted" at the end of _all_ toast notifications.  I don't think that's the intent.

Reply:
This seems to be some sort of caching with the toasts, because this method is only called for the delete flow and not "sync timeout" etc.

Reply:
It looks like `SetUpToast` is also called from `showToastNotification`, which is in turned called in [ItemListPresenter](https://github.com/mozilla-lockwise/lockwise-android/blob/297f829f468b6249ba6fc3691d84e2bdabfb1b01/app/src/main/java/mozilla/lockbox/presenter/ItemListPresenter.kt#L90).

Reply:
Orrrrr I accidentally switched the elvis values. Fixed now.